### PR TITLE
feat(containerbase): use install-tool for pipenv and hashin

### PR DESCRIPTION
**Pull request by viceice**
## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/manager/pipenv/__snapshots__/artifacts.spec.ts.snap [hunk 1]
```diff
@@ -226,7 +226,7 @@ exports[`modules/manager/pipenv/artifacts uses pipenv version from config 1`] =
     },
   },
   {
-    "cmd": "docker run --rm --name=renovate_sidecar --label=renovate_child -v "/tmp/github/some/repo":"/tmp/github/some/repo" -v "/tmp/renovate/cache":"/tmp/renovate/cache" -e PIPENV_CACHE_DIR -e BUILDPACK_CACHE_DIR -e CONTAINERBASE_CACHE_DIR -w "/tmp/github/some/repo" containerbase/sidecar bash -l -c "install-tool python 3.10.2 && pip install --user pipenv==2020.1.1 && pipenv lock"",
+    "cmd": "docker run --rm --name=renovate_sidecar --label=renovate_child -v "/tmp/github/some/repo":"/tmp/github/some/repo" -v "/tmp/renovate/cache":"/tmp/renovate/cache" -e PIPENV_CACHE_DIR -e BUILDPACK_CACHE_DIR -e CONTAINERBASE_CACHE_DIR -w "/tmp/github/some/repo" containerbase/sidecar bash -l -c "install-tool python 3.10.2 && install-tool pipenv ==2020.1.1 && pipenv lock"",
```
Comment (rarkins) [inline lib/modules/manager/pipenv/__snapshots__/artifacts.spec.ts.snap:229]:
```suggestion
    "cmd": "docker run --rm --name=renovate_sidecar --label=renovate_child -v "/tmp/github/some/repo":"/tmp/github/some/repo" -v "/tmp/renovate/cache":"/tmp/renovate/cache" -e PIPENV_CACHE_DIR -e BUILDPACK_CACHE_DIR -e CONTAINERBASE_CACHE_DIR -w "/tmp/github/some/repo" containerbase/sidecar bash -l -c "install-tool python 3.10.2 && install-tool pipenv 2020.1.1 && pipenv lock"",
```

The `==` need stripping, unless you do that in install-tool now?

Comment (viceice) [reply lib/modules/manager/pipenv/__snapshots__/artifacts.spec.ts.snap:229]: ups, yes.

**Review by rarkins — APPROVED** [review]

**Comment by rarkins** [comment]
@viceice most of these changes were de-duped. PR still look ok?

**Comment by viceice** [comment]
I need to check, but it seems only the hashin constraints are left in this PR. will fix it next week

**Comment by rarkins** [comment]
Easier to resubmit

**Pull request closed without merging**